dev-scripts-additional-config

```diff
@@ -0,0 +1,5 @@
+export IRONIC_LOCAL_IMAGE=https://github.com/openshift/ironic-image
```

Comment:
What is this file? Just an example?

Reply:
It's a new feature added to allow CI testing of a configuration change proposed in a PR (otherwise the new code couldn't be exercised). It is used by the `e2e-metal-ipi` [devscript setup step](https://github.com/openshift/release/blob/ac8d3dc95e028d6cc8e5abb5d4c61aad2a51ba17/ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh#L91-L94).